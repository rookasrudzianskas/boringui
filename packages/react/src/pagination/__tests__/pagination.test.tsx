import React from "react";
import {mount} from "enzyme";

import Pagination from "../pagination";

describe("Pagination", () => {
  it("should render correctly", () => {
    const wrapper = mount(<Pagination />);

    expect(wrapper.html()).toMatchSnapshot();
    expect(() => wrapper.unmount()).not.toThrow();
  });
});
