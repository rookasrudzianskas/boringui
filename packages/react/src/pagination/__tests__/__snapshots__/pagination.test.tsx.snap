// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagination should render correctly 1`] = `"<nav class=\\"nextui-c-eKVPvf nextui-c-eKVPvf-iBFhCV-color-default nextui-c-eKVPvf-UECOZ-size-md nextui-c-eKVPvf-iTJsWG-borderWeight-normal nextui-c-eKVPvf-ibzOHM-onlyDots-false nextui-c-eKVPvf-bNGYLg-rounded-false nextui-c-eKVPvf-QbeIJ-noMargin-false nextui-pagination\\"><button aria-label=\\"previous page button\\" class=\\"nextui-c-cUthvm nextui-c-huiNHE nextui-c-cUthvm-gZRAkC-disabled-true nextui-c-cUthvm-SWDEj-animated-true nextui-pagination-item nextui-pagination-item-animated nextui-pagination-item-disabled nextui-pagination-item-preserve-content\\" tabindex=\\"-1\\"><span class=\\"nextui-c-fItrmj nextui-pagination-item-content\\"><svg class=\\"nextui-c-eKuzCY nextui-c-eKuzCY-xROYZ-isPrev-true nextui-pagination-icon\\" fill=\\"none\\" focusable=\\"false\\" role=\\"presentation\\" viewBox=\\"0 0 24 24\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path d=\\"M15.5 19l-7-7 7-7\\" stroke=\\"currentColor\\" stroke-linecap=\\"round\\" stroke-linejoin=\\"round\\" stroke-width=\\"1.5\\"></path></svg></span></button><div aria-hidden=\\"true\\" class=\\"nextui-c-cAbbLF nextui-c-cAbbLF-QIzBs-animated-true nextui-c-cAbbLF-ikgtVxo-css nextui-pagination-highlight nextui-pagination-highlight--active\\" style=\\"--nextui--paginationLeft: calc(var(--nextui--paginationSize) * 1 + 6px);\\"></div><button aria-label=\\"1 item active\\" class=\\"nextui-c-cUthvm nextui-c-huiNHE nextui-c-cUthvm-dZWCtT-active-true nextui-c-cUthvm-SWDEj-animated-true nextui-pagination-item nextui-pagination-item-active nextui-pagination-item-animated\\" tabindex=\\"0\\"><span class=\\"nextui-c-fItrmj nextui-pagination-item-content\\">1</span></button><button aria-label=\\"next page button\\" class=\\"nextui-c-cUthvm nextui-c-huiNHE nextui-c-cUthvm-gZRAkC-disabled-true nextui-c-cUthvm-SWDEj-animated-true nextui-pagination-item nextui-pagination-item-animated nextui-pagination-item-disabled nextui-pagination-item-preserve-content\\" tabindex=\\"-1\\"><span class=\\"nextui-c-fItrmj nextui-pagination-item-content\\"><svg class=\\"nextui-c-eKuzCY nextui-pagination-icon\\" fill=\\"none\\" focusable=\\"false\\" role=\\"presentation\\" viewBox=\\"0 0 24 24\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path d=\\"M15.5 19l-7-7 7-7\\" stroke=\\"currentColor\\" stroke-linecap=\\"round\\" stroke-linejoin=\\"round\\" stroke-width=\\"1.5\\"></path></svg></span></button></nav>"`;
